RevenueCatUI/Templates/V2/PaywallsV2View.swift

```diff
@@ -237,23 +237,47 @@ private struct LoadedPaywallsV2View: View {
     }
 
     var body: some View {
-        VStack(spacing: 0) {
-            ComponentsView(
-                componentViewModels: [paywallState.componentViewModel],
-                onDismiss: self.onDismiss
+        GeometryReader { proxy in
+            VStack(spacing: 0) {
+                ComponentsView(
+                    componentViewModels: [.root(paywallState.rootViewModel)],
+                    onDismiss: self.onDismiss
+                )
+            }
+            // If the first view in the first stack is an image,
+            // we will ignore safe area pass the safe area insets in to environment
+            // If the image is in a ZStack, the ZStack will push non-images
+            // down with the inset
+            .applyIf(paywallState.rootViewModel.firstImageInfo != nil, apply: { view in
+                view
+                    .environment(\.safeAreaInsets, proxy.safeAreaInsets)
+                    .edgesIgnoringSafeArea(.top)
+            })
+            // This is normal
```

Comment:
What does this comment mean?

Reply:
Ok. I see you removed it in #4746 😄

Reply:
Ha! This was a note to me when testing this but gets fixed in a follow up PR